reports/README.md

```diff
@@ -16,10 +16,16 @@ make generate_parameters
 make all -j 8
 ```
 
-Finally, to push report data to the production bucket, run the following.
-
-```
-# NOTE: currently need to replace gtfs-data-test with gtfs-data in Makefile
-# to push to production
-make sync
-```
+## Using Postmark to send emails (SAFELY)
+We are currently using the postmark API to send the reports via email to their respective clients
+1. Access to Postmark (contact for access to Bitwarden)
+    There is a Cal-ITP developer shared" collection in Bitwarden that contains keys for the reports sandbox server. An indivudal will not be given access to prod unless it is absolutely necessary. 
```

Comment:
These line breaks don't end up being rendered, so it looks like a run-on sentence.